lib/models/forum_post.dart

```diff
@@ -0,0 +1,25 @@
+class ForumPost {
+  final int fid;
+  final Map<String, String> title;
+  final Map<String, String> description;
```

Comment:
should models have map title/desc fields or string ? should the app query the api again when the language is changed or just change the key to retrieve a diff val from the map ?

Reply:
changed to be language specific